tensorflow_addons/image/transform_test.py

```diff
@@ -157,130 +155,120 @@ def test_zeros(self):
+                    np.pi / 2, 4, 4)
                 # Translate right by 1 (the transformation matrix is always inverted,
                 # hence the -1).
-                translation = constant_op.constant([1, 0, -1,
-                                                    0, 1, 0,
-                                                    0, 0],
+                translation = constant_op.constant([1, 0, -1, 0, 1, 0, 0, 0],
                                                    dtype=dtypes.float32)
-                composed = image_ops.compose_transforms(rotation, translation)
-                image_transformed = image_ops.transform(image, composed)
-                self.assertAllEqual(image_transformed,
-                                    [[0, 0, 0, 0],
-                                        [0, 1, 0, 1],
-                                        [0, 1, 0, 1],
-                                    [0, 1, 1, 1]])
+                composed = transform_ops.compose_transforms(
+                    rotation, translation)
+                image_transformed = transform_ops.transform(image, composed)
+                self.assertAllEqual(
+                    image_transformed,
+                    [[0, 0, 0, 0], [0, 1, 0, 1], [0, 1, 0, 1], [0, 1, 1, 1]])
 
     def test_bilinear(self):
         with self.cached_session():
             image = constant_op.constant(
-                [[0, 0, 0, 0, 0],
-                 [0, 1, 1, 1, 0],
-                 [0, 1, 0, 1, 0],
-                 [0, 1, 1, 1, 0],
-                 [0, 0, 0, 0, 0]],
-                dtypes.float32)
+                [[0, 0, 0, 0, 0], [0, 1, 1, 1, 0], [0, 1, 0, 1, 0],
+                 [0, 1, 1, 1, 0], [0, 0, 0, 0, 0]], dtypes.float32)
             # The following result matches:
             # >>> scipy.ndimage.rotate(image, 45, order=1, reshape=False)
             # which uses spline interpolation of order 1, equivalent to bilinear
             # interpolation.
             self.assertAllClose(
-                image_ops.rotate(image, np.pi / 4.0, interpolation="BILINEAR"),
+                transform_ops.rotate(
+                    image, np.pi / 4.0, interpolation="BILINEAR"),
                 [[0.000, 0.000, 0.343, 0.000, 0.000],
                  [0.000, 0.586, 0.914, 0.586, 0.000],
                  [0.343, 0.914, 0.000, 0.914, 0.343],
                  [0.000, 0.586, 0.914, 0.586, 0.000],
                  [0.000, 0.000, 0.343, 0.000, 0.000]],
                 atol=0.001)
             self.assertAllClose(
-                image_ops.rotate(image, np.pi / 4.0, interpolation="NEAREST"),
-                [[0, 0, 1, 0, 0],
-                 [0, 1, 1, 1, 0],
-                 [1, 1, 0, 1, 1],
-                 [0, 1, 1, 1, 0],
-                 [0, 0, 1, 0, 0]])
+                transform_ops.rotate(
+                    image, np.pi / 4.0, interpolation="NEAREST"),
+                [[0, 0, 1, 0, 0], [0, 1, 1, 1, 0], [1, 1, 0, 1, 1],
```

Comment:
Could you disable auto-formatting data here by

```python
            # yapf: disable
            self.assertAllClose(
                image_ops.rotate(image, np.pi / 4.0, interpolation="NEAREST"),
                [[0, 0, 1, 0, 0],
                 [0, 1, 1, 1, 0],
                 [1, 1, 0, 1, 1],
                 [0, 1, 1, 1, 0],
                 [0, 0, 1, 0, 0]])
            # yapf: enable
```

see https://github.com/tensorflow/addons/blob/master/STYLE_GUIDE.md#python for more details :-)

Reply:
Please fix the format of all data before, sorry for the incovenience.

Reply:
Don't forget the last comment `# yapf: enable`, Kyle :-)

Reply:
done